sql/connect/server/src/main/scala/org/apache/spark/sql/connect/service/SparkConnectServer.scala

```diff
@@ -28,7 +29,11 @@ object SparkConnectServer extends Logging {
   def main(args: Array[String]): Unit = {
     // Set the active Spark Session, and starts SparkEnv instance (via Spark Context)
     logInfo("Starting Spark session.")
-    val session = SparkSession.builder().getOrCreate()
+    val session = SparkSession
+      .builder()
+      .config(SQLConf.ARTIFACTS_SESSION_ISOLATION_ENABLED.key, true)
```

Comment:
This is the default right?

Reply:
Yes. I removed this line.

Reply:
What did you remove?

Reply:
I removed setting this flag before but some connect tests are failing afterwards, seems not to get the default value. Now I added it back to make sure the isolation is indeed enabled.